tasks/eva_2461/correct_contig_error_in_study.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+import hashlib
+import logging
+import traceback
+
+import pymongo
+from ebi_eva_common_pyutils.mongodb import MongoDatabase
+
+
+def get_SHA1(variant_rec):
+    """Calculate the SHA1 digest from the seq, study, contig, start, ref, and alt attributes of the variant"""
+    h = hashlib.sha1()
+    keys = ['seq', 'study', 'contig', 'start', 'ref', 'alt']
+    h.update('_'.join([str(variant_rec[key]) for key in keys]).encode())
+    return h.hexdigest().upper()
+
+
+def replace_with_correct_contig(mongo_source):
+    correct_contig = 'AF034253.1'
+    sve_collection = mongo_source.mongo_handle[mongo_source.db_name]["submittedVariantEntity"]
+    filter_criteria = {'seq': 'GCA_000003025.4', 'study': 'PRJEB43246', 'contig': 'M'}
+    cursor = sve_collection.find(filter_criteria)
```

Comment:
Even though we know the number of documents is small,  Mongo is trying to find a needle in a haystack, I would suggest providing a parameter for cursor timeout with a generous value here.

Reply:
I would advise to pass `no_cursor_timeout=True` see [doc](https://pymongo.readthedocs.io/en/stable/api/pymongo/collection.html#pymongo.collection.Collection.find)

Reply:
updated